src/main/java/co/rsk/federate/signing/config/SignerConfig.java

```diff
@@ -1,6 +1,6 @@
 /*
  * This file is part of RskJ
- * Copyright (C) 2018 RSK Labs Ltd.
+ * Copyright (C) 2024 RSK Labs Ltd.
```

Comment:
Not sure what to do here. Should we append this license header file to all files? cc: @marcos-iov

Reply:
Done